fetch-todolist/src/shared/FormikControl.js

```diff
@@ -0,0 +1,18 @@
+import {} from "formik";
+import DatePicker from "./DatePicker";
+import Input from "./Input";
+import Textarea from "./Textarea";
+const FormikControl = (props) => {
+  const { control, ...rest } = props;
+  switch (control) {
+    case "input":
```

Comment:
cases values can be fetched for constants

Reply:
added a file for constants